.doc_gen/metadata/bedrock-runtime_metadata.yaml

```diff
@@ -692,6 +700,14 @@ bedrock-runtime_ConverseStream_MetaLlama:
             - description: Send a text message to Meta Llama, using Bedrock's Converse API and process the response stream in real-time.
               snippet_tags:
                 - javascript.v3.bedrock-runtime.ConverseStream_MetaLlama
+    Swift:
+      versions:
+        - sdk_version: 3
```

Comment:
Is version 3 correct ?  Line 420 says version 1. I don't know which version is correct but the two should be the same